libs/dh/metering-point/feature-overview/src/components/measurement/dh-measurements.component_v2.ts

```diff
@@ -89,10 +99,12 @@ export class DhMeasurementsV2Component {
       position: {
         accessor: null,
         cell: (value) => (this.measurements().findIndex((x) => x === value) + 1).toString(),
+        footer: { value: signal(this.transloco.translate('meteringPoint.measurements.sum')) },
```

Comment:
question: didn't Transloco support a Signal-based translation? Can it be used here?

Reply:
I tried, but ran into the "Can not run without a context error"